package.json

```diff
@@ -36,6 +36,7 @@
     "posix": "^4.0.0"
   },
   "devDependencies": {
+    "eslint-config-digitalbazaar": "^1.7.0",
```

Comment:
need to add eslint as well per convention.

Reply:
oh really? it works fine with out. added it anyways.